stories/Undo.stories.tsx

```diff
@@ -17,9 +42,153 @@ export default {
     Remove,
     Add
   }
+} as Meta;
+
+type Props = {}
+
+export const Simple: Story<Props> = () => {
```

Comment:
Can you just omit `<Props>` and it still work?

Reply:
Yeah it'll still work, it's a habit of mine, makes it friendlier to add props faster, but I can remove it if you think it's unnecessary boilerplate code.

Reply:
Ya - I'd rather keep it simple.